test/expressions.tests.ts

```diff
@@ -3813,7 +3813,7 @@ private static readonly Parser<Node> NodeParser =
           Token.Punctuation.Comma,
           Token.Variables.ReadWrite("Value"),
           Token.Operators.Assignment,
-          Token.Variables.ReadWrite("value"),
```

Comment:
I could make it so that `Variables.ReadWrite("value")` returns `Variables.Value` (possibly with a deprecated mark with "value" as argument type).
Would that be necessary/efficient for future writing tests?

Reply:
No need to, new tests should just reference `Variables.Value` directly.